### PR TITLE
web-172 - Show CFP deadline in the introduction

### PR DESCRIPTION
**Pull request by estyxx**

### CODE REVIEW

File: frontend/src/pages/[lang]/cfp/is-cfp-open.graphql [hunk 1]
```diff
@@ -2,5 +2,9 @@ query IsCfpOpen($conference: String!) {
   conference(code: $conference) {
     id
     isCFPOpen
+    deadlines {
```
Comment (marcoacierno) [inline frontend/src/pages/[lang]/cfp/is-cfp-open.graphql:5]: you can fetch a single deadline: https://github.com/pythonitalia/pycon/blob/5d0cac9834203f96204e423eb0b797c9704c041a/frontend/src/pages/index.graphql#L29

Comment (estyxx) [reply frontend/src/pages/[lang]/cfp/is-cfp-open.graphql]: Oh yeah, of course, I didn't see it!

File: frontend/src/pages/cfp/index.tsx [hunk 2]
```diff
@@ -57,18 +58,24 @@ const CfpSectionOrClosedMessage: React.SFC<{ open: boolean }> = ({ open }) => {
...
-
   return (
     <Fragment>
       <FormattedMessage id="cfp.pageTitle">
         {(text) => <MetaTags title={text} />}
       </FormattedMessage>
-      <Introduction />
+      <Introduction
+        deadline={formatDeadlineDate(
+          data?.conference.cfpDeadline?.end,
```
Comment (marcoacierno) [inline frontend/src/pages/cfp/index.tsx:75]: this doesn't seem correct, if we pass undefined `new Date()` would fail 🤔

Comment (marcoacierno) [reply frontend/src/pages/cfp/index.tsx:75]: image
this is what happens 🤔  we need to improve this CFP page after we merge my PR #2540

**Pull request merged**